src/ast/passes/resolve_imports.h

```diff
@@ -99,6 +99,7 @@ class Imports : public ast::State<"imports"> {
 };
 
 // This pass resolves imports from the AST itself.
-Pass CreateResolveImportsPass(std::vector<std::string> &&import_paths = {});
+Pass CreateResolveImportsPass(std::vector<std::string> &&import_paths = {},
+                              bool disable_stdlib = false);
```

Comment:
```suggestion
                              bool disable_stdlib_tests_only = false);
```
I know I suggested this name but maybe we make it more explicit.